### DISCUSSION TITLE
Shortcut from HASS directly to MASS with specific speaker selected

### DISCUSSION THREAD

**Discussion by ranoydegraaf** [music-assistant/support#877] [Feature requests and ideas]
I use the tap action on the media player card on my dashboard to get into Music Assistant, but it'll always hass the last used speaker selected for playback.

It would be great to add some sort of code to the tap action so that the speaker of that media player card is automatically selected when getting into Music Assistant.

**Comment by OzGav** [comment]
Relates to v1
